README.md

```diff
@@ -48,6 +54,25 @@ docker run --rm -v "$PWD/tmp/000017_0.jsonl:/mydata/data.jsonl" \
            post -c ess /mydata/data.jsonl
 ```
 
+### queries v2
+
```

Comment:
What feature provides `queries v2`? Could you add a short description? I've been in at the Confluence page, but there are only SQL queries, without any context.

Reply:
Yup, Hive queries. Their output is transformed by this script.